### PR TITLE
Further extend our services implementations

### PR DESCRIPTION
**Pull request by bylaws**
This pr implements all the services needed to boot Mario Odyssey, however it will still fail later on due to an unknown bug.

### CODE REVIEW

File: app/src/main/cpp/skyline/services/am/controller/ISelfController.cpp [hunk 1]
```diff
@@ -12,6 +12,8 @@ namespace skyline::service::am {
         {0xB, SFUNC(ISelfController::SetOperationModeChangedNotification)},
         {0xC, SFUNC(ISelfController::SetPerformanceModeChangedNotification)},
         {0xD, SFUNC(ISelfController::SetFocusHandlingMode)},
+        {0xE, SFUNC(ISelfController::SetRestartMessageEnabled)},
+
```
Comment (PixelyIon) [inline app/src/main/cpp/skyline/services/am/controller/ISelfController.cpp:16]: Extra spacing that isn't required

File: app/src/main/cpp/skyline/services/pctl/IParentalControlService.cpp [hunk 2]
```diff
@@ -0,0 +1,9 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright © 2020 Skyline Team and Contributors (https://github.com/skyline-emu/)
+
+#include "IParentalControlService.h"
+
+namespace skyline::service::pctl {
+    IParentalControlService::IParentalControlService(const DeviceState &state, ServiceManager &manager) : BaseService(state, manager, Service::pctl_IParentalControlService, "pctl:IParentalControlService", {
+    }) {}
```
Comment (PixelyIon) [inline app/src/main/cpp/skyline/services/pctl/IParentalControlService.cpp:8]: Sorry, but I just noticed that this wasn't due to line wrapping but was on the next line. We don't tend to put the `}) {}` on the next line if the list is empty, only if it's being terminated while having 1 or more item.

Comment (bylaws) [reply app/src/main/cpp/skyline/services/pctl/IParentalControlService.cpp:8]: Meeeehhhh, it's easier to have it two lines for the future :(

Comment (PixelyIon) [reply app/src/main/cpp/skyline/services/pctl/IParentalControlService.cpp:8]: Fine then, I don't mind it all that much :)

**Pull request merged**